images/virtualization-artifact/pkg/controller/vmbda/internal/watcher/kvvmi_watcher.go

```diff
@@ -127,7 +123,7 @@ func (eh KVVMIEventHandler) getHotPluggedVolumeStatuses(obj client.Object) map[s
 			var name string
 			name, ok = kvbuilder.GetOriginalDiskName(vs.Name)
 			if !ok {
-				eh.logger.Warn("VolumeStatus is not a Disk", "vsName", vs.Name, "name", kvvmi.Name, "ns", kvvmi.Namespace)
+				slog.Default().Warn("VolumeStatus is not a Disk", "vsName", vs.Name, "name", kvvmi.Name, "ns", kvvmi.Namespace)
```

Comment:
Why usr default logger in this case? A common question about the watcher package

Reply:
watcher doesn't have ctx to extract logger from, it's easier to use default logger here